services/src/floodlight-modules/src/main/java/org/openkilda/floodlight/switchmanager/SwitchManager.java

```diff
@@ -807,6 +825,40 @@ private long buildAndinstallMeter(final IOFSwitch sw, final DatapathId dpid, fin
         return pushFlow(sw, "--InstallMeter--", meterMod);
     }
 
+    @VisibleForTesting
+    long installMeterForDefaultRule(IOFSwitch sw, long cookie, boolean isBroadcast) throws OfInstallException {
```

Comment:
1. We can reuse `.buildAndinstallMeter(...)`... it can be altered to accept flags and called from here.

2. `boolean isBroadcast`... this is extremely ugly way to define what we should do... Please do not spread this approach further. You can pass bandwidth value from `installVerificationRule` it is already have this ugly argument/condition... so let it be the only place we use it.